jaeger/client.go

```diff
@@ -17,11 +15,19 @@ import (
 type ClientInterface interface {
 	GetJaegerServices() (services []string, code int, err error)
 	GetSpans(namespace, service, startMicros, endMicros string) ([]Span, error)
-	GetTraces(namespace string, service string, rawQuery string) (traces []jaegerModels.Trace, code int, err error)
-	GetTraceDetail(traceId string) (trace []jaegerModels.Trace, code int, err error)
+	GetJaegerInfo() (*JaegerInfo, int, error)
+	GetTraces(namespace string, service string, rawQuery string) (traces *JaegerResponse, code int, err error)
+	GetTraceDetail(traceId string) (trace *JaegerResponse, code int, err error)
 	GetErrorTraces(ns string, srv string) (errorTraces int, err error)
 }
 
+type JaegerInfo struct {
+	URL                string `json:"url"`
+	NamespaceSelector  bool   `json:"namespace_selector"`
```

Comment:
we use camel case for json (unless there's a special reason to do otherwise) ; should be changed here?

Reply:
changed I forgot it